core/src/main/scala/org/apache/spark/sql/rapids/tool/profiling/ApplicationInfo.scala

```diff
@@ -346,6 +346,86 @@ class ApplicationInfo(
     }
   }
 
+  /**
+   * This function is meant to clean up Delta log execs so that you could align
+   * SQL ids between CPU and GPU eventlogs. It attempts to remove any delta log
+   * only SQL ids. This includes reading checkpoints, delta_log json files,
+   * updating Delta state cache/table. There is also one special case for
+   * MergeIntoCommandEdge that I think showed up because some eventlogs might not
+   * be on the exact same data.
+   */
+  def cleanSQLIdsToAlign(): Seq[SQLCleanAndAlignIdsProfileResult] = {
+    val noDeltaLogs = sqlPlans.filter {
+      case (id, planInfo) =>
+        val planGraph = ToolsPlanGraph(planInfo)
+        val sqlDesc = sqlIdToInfo(id).physicalPlanDesc
+        val leftoverNodes = planGraph.allNodes.filter { node =>
+          val normalizedNodeName = node.name.stripSuffix("$")
+          normalizedNodeName match {
+            case "RDDScanExec" =>
+              if (sqlDesc.contains("Delta Table State") ||
+                sqlDesc.contains("Delta Table Checkpoint") ||
+                sqlDesc.contains("delta_log")) {
+                false
+              } else {
+                true
+              }
+            case s if s.contains("WholeStageCodegen") =>
+              if (sqlDesc.contains("_delta_log")) {
+                false
+              } else {
+                true
+              }
+            case s if s.contains("MergeIntoCommandEdge") =>
+              // this is a bit odd but GPU doesn't accelerate anyway, I think this might be
```

Comment:
Double checking to assess the `MergeIntoCommandEdge`. Does the comment mean that Exec is supported but the GPU does not achieve any Speedups?
For Qualification, we mark the exec as supported and we use DataWritingExecCommand as its acceleration speed.